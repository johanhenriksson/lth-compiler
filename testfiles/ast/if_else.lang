int main() {
    if (false) {
    } else {
    }
}
