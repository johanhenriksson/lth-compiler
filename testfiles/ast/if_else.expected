Program
  List
    Function
      TypeId
      IdDecl
      List
      Block
        List
          IfStmt
            FalseLiteral
            Block
              List
            Opt
              Block
                List
